### PR TITLE
Raidboss: add more ja/cn translation in emerald_weapon-ex

### PR DESCRIPTION
**Pull request by MaikoTan**
@r4ha Do you have time to review it?
I am not very confident with these tho.

### CODE REVIEW

**Comment by r4ha** [comment]
> @r4ha Do you have time to review it?
> I am not very confident with these tho.

sure!

File: ui/raidboss/data/05-shb/trial/emerald_weapon-ex.js [hunk 1]
```diff
@@ -345,14 +356,14 @@ export default {
...
       'locale': 'ja',
-      'missingTranslations': true,
       'replaceSync': {
         'Black Wolf\'s Image': 'ガイウスの幻影',
         'Imperial Image': '帝国兵の幻影',
         'Reaper Image': 'リーパーの幻影',
         'The Emerald Weapon': 'エメラルドウェポン',
       },
       'replaceText': {
+        '--cutscene--': '--カットシン--',
```
Comment (r4ha) [inline ui/raidboss/data/05-shb/trial/emerald_weapon-ex.js:366]: '--cutscene--': '--カットシーン--',

**Review by r4ha — CHANGES_REQUESTED** [review]

File: ui/raidboss/data/05-shb/trial/emerald_weapon-ex.js (hunk 1 above)
Comment (MaikoTan) [reply ui/raidboss/data/05-shb/trial/emerald_weapon-ex.js:366]: Oh?
Is this long tone? So I always spell it wrong?

Comment (r4ha) [reply ui/raidboss/data/05-shb/trial/emerald_weapon-ex.js]: maybe...?

**Pull request merged**